dotcom-rendering/src/lib/articleCount.ts

```diff
@@ -37,15 +39,22 @@ export const getArticleCounts = async (
 	// hasOptedOut needs to be done before we check if articleCount is set in the window
 	// This is because a potential race condition where one invocation of getArticleCounts
 	// is waiting for hasOptedOut another invocation might receive it and increment the article count.
+
+	const keywordAndToneTagIds: string[] = [
+		...keywordIds.split(','),
+		...tags
+			.filter((tag) => tag?.type.toLowerCase() === 'tone')
```

Comment:
Just noticed that if we change this to:
`['tone', 'keyword'].includes(tag?.type.toLowerCase())`
then we can avoid passing around `keywordIds`, which is just based on the keyword tags.

Reply:
I have removed the keywordId and now using only tags which has both keyword and tone types.